src/res/translation/translation_zh_CN.ts

```diff
@@ -0,0 +1,3072 @@
+        <source>&amp;Open </source>
+        <translation>打开(&amp;O)</translation>
+    </message>
+    <message>
+        <location filename="../../serverdlg.cpp" line="341"/>
+        <source> Server</source>
+        <translation> 服务器</translation>
+    </message>
+    <message>
+        <location filename="../../serverdlg.cpp" line="344"/>
+        <source>&amp;Window</source>
+        <translation>窗口(&amp;W)</translation>
+    </message>
+    <message>
+        <location filename="../../serverdlg.cpp" line="574"/>
+        <source>Select Main Recording Directory</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../serverdlg.h" line="56"/>
+        <source>Recording</source>
+        <translation>录制中</translation>
+    </message>
+    <message>
+        <location filename="../../serverdlg.h" line="55"/>
+        <source>Not recording</source>
+        <translation>未在录制</translation>
+    </message>
+    <message>
+        <location filename="../../serverdlg.h" line="53"/>
+        <source>Not initialised</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../serverdlg.h" line="54"/>
+        <source>Not enabled</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../util.h" line="594"/>
+        <source>Unregistered</source>
+        <translation>未登记</translation>
+    </message>
+    <message>
+        <location filename="../../util.h" line="597"/>
+        <source>Bad address</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../util.h" line="600"/>
+        <source>Registration requested</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../util.h" line="603"/>
+        <source>Registration failed</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../util.h" line="606"/>
+        <source>Check server version</source>
+        <translation>检查服务器版本</translation>
+    </message>
+    <message>
+        <location filename="../../util.h" line="609"/>
+        <source>Registered</source>
+        <translation>已登记</translation>
+    </message>
+    <message>
+        <location filename="../../util.h" line="612"/>
+        <source>Directory Server full</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../util.h" line="615"/>
+        <source>Your server version is too old</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../util.h" line="618"/>
+        <source>Requirements not fulfilled</source>
```

Comment:
Tagging @gilgongo here. I doubt this error message is user friendly... Should we open an issue on that?

Reply:
Seems like we should re-write that. Wonder what it refers to?

Reply:
Probably if the directory rejects the server to register with it?